index.js

```diff
@@ -1,43 +1,35 @@
 const express = require("express")
 const app = express()
 const path = require("path")
+const { cryptoPairs } = require("./seed/seed")
 
-// requiring fetch (node-fetch)
+// requiring fetch (node-fetch) instead of using import
 const fetch = (...args) =>
   import("node-fetch").then(({ default: fetch }) => fetch(...args))
 
-app.set("view engine", "ejs") // set ejs as view engine
+app.set("view engine", "ejs")
 app.set("views", path.join(__dirname, "views"))
 app.use(express.static(path.join(__dirname, "public")))
 
-const urls = [
-  "https://api.cryptonator.com/api/ticker/btc-usd",
-  "https://api.cryptonator.com/api/ticker/eth-usd",
-  "https://api.cryptonator.com/api/ticker/ada-usd",
-]
+let cryptoData = {}
```

Comment:
Did you know that you can define objects and Arrays almost always directly as `const` even when you change the values inside? If you didn't find out why :D

```suggestion
const cryptoData = {}
```

Reply:
Yes! Something I learned in the advanced JS course (that I'm still taking) is that Objects and Arrays are passed by reference, as such changing _what_ they're storing doesn't affect their place in memory. Problems/bugs, and some of the ensuing confusion it can cause, can arise when someone attempts to re-assign or re-declare something they shouldn't.




I think.